library/ssl_misc.h

```diff
@@ -2273,7 +2274,7 @@ static inline int mbedtls_ssl_tls13_sig_alg_for_cert_verify_is_supported(
     const uint16_t sig_alg)
 {
     switch (sig_alg) {
-#if defined(MBEDTLS_ECDSA_C)
+#if defined(MBEDTLS_PK_CAN_ECDSA_SOME)
```

Comment:
Note: I'm not sure `_SOME` is the right one here, but since we're not aiming to support builds with only sign or only verify now or in the near future, that's OK. (When there is a relatively obvious right choice between some, sign, verify, both, we should try to do the right thing even if it isn't tested now, but when it's not obvious, I don't think we need to think too hard about it.)